runtests.py

```diff
@@ -198,35 +210,34 @@ def run_test_suite(args):
         cov.save()
 
         if percentage < TESTS_THRESHOLD:
-            sys.stderr.write(colored(text="YOUR CHANGES HAVE CAUSED TEST COVERAGE TO DROP. " +
-                                     "WAS {old}%, IS NOW {new}%.\n\n".format(old=TESTS_THRESHOLD, new=percentage),
+            sys.stderr.write(colored(text="Your changes have caused test coverage to drop. " +
```

Comment:
Upper case just generally has an angrier tone so I suggest lowercase for less anxiety. Also upper case just seems out of place.